conans/test/unittests/client/generators/b2_test.py

```diff
@@ -298,7 +298,7 @@ def b2_test(self):
 # mypkg
 if $(__define_targets__) {
     call-in-project $(mypkg-mod) : lib MyLib1
-        :
+        : ''' + '''
```

Comment:
The reason for this change is to protect the trailing space on this line. My editor is configured to remove trailing whitespace on saves. At first I wanted to temporally disable this feature in the editor, but then I thought, that these days it's quite a popular feature, so a different person may inadvertently remove that space character.
The space itself is important, because the respective line in generator's template is `: {libs}` and produces a trailing space if `libs` is empty.